src/cli.test.ts

```diff
@@ -0,0 +1,128 @@
+import { execSync } from 'node:child_process';
+
+import { goSync } from '@api3/promise-utils';
+
+const CLI_EXECUTABLE = 'dist/src/cli.js';
+
+type CommandArg = [string, string | number | boolean];
+
+describe('cli tests', () => {
+  const execCommand = (command: string, ...args: CommandArg[]) => {
+    const quote = (val: string) => `"${val}"`;
```

Comment:
I think you should also get rid of `quote()` considering that we're not recommending the user to enclose argument values in quotation marks anywhere. That was an airnode-admin thing where seed phrases needed to be enclosed because they have spaces in them.